src/rateEngine/utils/assumptions.ts

```diff
@@ -0,0 +1 @@
+export const DAYS_PER_MONTH = [31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31];
```

Comment:
as you mentioned above, this is actually not accurate for leap years... should we make this a function that accepts a year? and do we actually have an existing leap year bug in the billing determinants that use this?